pkgs/shells/fish/plugins/gruvbox.nix

```
@@ -0,0 +1,17 @@
+{ lib , buildFishPlugin , fetchFromGitHub }:
+buildFishPlugin {
+  pname = "gruvbox";
+  version = "unstable-2021-10-12";
```

```suggestion
  version = "0-unstable-2021-10-12";
```

recent policy is that versions must start with a digit.